pypit_files/shane_kast_red_600_7500_d55.pypit

```diff
@@ -1,27 +1,27 @@
 # This is a comment line
 
 # Change the default settings
-run ncpus 1
-run spectrograph shane_kast_red_ret
-output verbosity 2
-output overwrite True
-output sorted kast_red_600_7500_d55
+[rdx]
+    spectrograph = shane_kast_red_ret
+    pipeline = ARMS
+    sortroot = kast_red_600_7500_d55
 
-# Reduce
-#arc calibrate IDpixels 220.63808792,417.17122925,562.95948074,669.07189712,884.38191292
-#arc calibrate IDwaves 5868.4165,6334.4276,6678.2766,6929.4672,7438.8981
-trace slits tilts params [1,1,1]
-trace slits tilts method spca
+[calibrations]
+    [[biasframe]]
+        number = 3
+    [[pixelflatframe]]
+        number = 3
+    [[standardframe]]
+        useframe = r136.fits.gz
+    [[tilts]]
+        method = spca
+        params = 1, 1, 1
 
 # Read in the data
 data read
  ~/Dropbox/PYPIT/TEST_SUITES/Kast_red/600_7500_d55/r1*.fits.gz
```

Comment:
Do we still distribute via dropbox?

Reply:
These paths get overwritten by `pypit_test`.  This is a question for @profxj, I think.

Reply:
They are over-written